### ISSUE TITLE
Tabs control: tabpanel needs to be in tab order if there's no focusable tabpanel content

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon#7078]
Steps:
- Open the [Tabs control example](https://carbon-components-react.netlify.app/iframe.html?id=tabs--default)
- Type the <kbd>Tab</kbd> key - focus moves to "Tab Label 2"
- Type the <kbd>Tab</kbd> key again - focus should move into the tabpanel (i.e. the div containing "Content for second tab...") but instead, it goes to the browser's location bar.

The [APG Guidance for Tabs component](https://w3c.github.io/aria-practices/#keyboard-interaction-20) says that if focus is in the `tablist` and the user types the <kbd>Tab</kbd> key:
> ... move focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable.

The Carbon Tabs component doesn't need to know what focusable element inside the tabpanel contains meaningful content. :)
It only needs to know whether or not _any_ of the `tabpanel` content is focusable. If none are, then the `tabpanel` itself needs to be focusable (i.e. have `tabindex="0"`).

Here's how one Tabs component decided on a prop for this: point 2 in https://github.com/juanca/react-aria-components/pull/6
based on this comment in issue https://github.com/w3c/aria-practices/issues/323#issuecomment-400011586.

Happy to talk this through before anything is coded.

**Comment by dakahn** [comment]
Wow, that's a really good catch. The use case you describe in https://github.com/w3c/aria-practices/issues/323#issuecomment-400011586 is totally reasonable. Thank you so much for bringing this to our attention!
